pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -0,0 +1,241 @@
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+// ReconcileSCCRecommendedName is the recommended command name
+const ReconcileSCCRecommendedName = "reconcile-sccs"
+
+type ReconcileSCCOptions struct {
+	Confirmed bool
```

Comment:
this needs more switches, since this object combines grants and bindings. I would expect to be able to reconcile the following things independently:

1. SCC definitions
2. SCC priorities
3. SCC subjects, additive only
4. SCC subjects, replace